src/Behat/Behat/Snippet/Printer/ConsoleSnippetPrinter.php

```diff
@@ -57,13 +57,20 @@ public function __construct(OutputInterface $output, TranslatorInterface $transl
      */
     public function printSnippets($targetName, array $snippets)
     {
-        $message = $this->translator->trans('snippet_proposal_title', array('%count%' => $targetName), 'output');
+        $message = $this->translator->trans('snippet_proposal_title', ['%count%' => $targetName], 'output');
 
         $this->output->writeln('--- ' . $message . PHP_EOL);
 
+        $usedClasses = [];
         foreach ($snippets as $snippet) {
+            foreach ($snippet->getUsedClasses() as $usedClass) {
+                $usedClasses[$usedClass] = true;
+            }
+
             $this->output->writeln(sprintf('<snippet_undefined>%s</snippet_undefined>', $snippet->getSnippet()) . PHP_EOL);
         }
+
+        $this->outputClassesUsesStatements($usedClasses);
```

Comment:
Slightly cleaner IMO and solves that I think the PHPDoc typehint ` * @param array<string> $usedClasses` below is not currently correct as it's actually getting `array<string,true>` at the moment?

```suggestion
        $this->outputClassesUsesStatements(array_keys($usedClasses));
```